arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -339,6 +339,13 @@ config CORTEX_M_DWT
 config TEST_EXTRA_STACK_SIZE
 	default 512 if TEST_ARM_CORTEX_M && FPU_SHARING
 
+config TRAP_UNALIGNED_ACCESS
+	bool "Enable unaligned access trap"
```

Comment:
Since this option is not really configurable on ARMv6-M and ARMv8-M Baseline cores, this should depend on `!ARMV6_M_ARMV8_M_BASELINE`.

Reply:
Cool, done.